views/js/controller/runtime/testRunner.js

```diff
@@ -956,6 +994,35 @@ define([
             },
 
             /**
+             * Processes an error
+             * @param {Object} error
+             */
+            processError : function processError(error) {
+                var self = this;
+
+                // keep disabled
+                this.hideGui();
+                this.beforeTransition();
+
+                // ask the parent to display a message
+                iframeNotifier.parent('alertMessage', {
```

Comment:
Real question : I am not about the event naming convention, should we not avoid upper case ? should we have alert-message or message-alert instead ?